assets/css/app.css

```
@@ -0,0 +1,46 @@
+/* ボタンの見せ方 */
+button,
+.button {
+  border-radius: 0.5rem;
+  border: solid 4px #555;
+  padding: 0.4em 1em;
+  cursor: pointer;
+  display: inline-block;
+  background-color: #fff;
+  color: #555;
+  text-decoration: none;
+}
+
+button:hover,
+.button:hover {
+  transform: translate(-1px, -1px);
+  box-shadow: 1px 1px 0 #555;
+}
+
+button:active,
+.button:active {
+  transform: translate(1px, 1px);
+  box-shadow: none;
+}
+
+/* ポケモンをつまえるボタン */
+.button .pokemon-ball {
+  width: 36px;
+  height: 36px;
+}
+
+.button:hover .pokemon-ball {
+  animation: bounce;
+  animation-duration: 0.8s;
+}
+
+/* リストのスタイル */
+.gamify-item:last-child img {
```

```suggestion
.gamify-item:hover > img, 
.gamify-item:focus-within > img {
```

選択中のポケモンがはねてくれるといいなと思いましたがいかがでしょう

---

delayの調整とかも必要かもしれません（おまかせします）

---

実装しました
https://github.com/webdino/lyceum-pokemon/pull/40#discussion_r819343021